yt/frontends/ramses/particle_handlers.py

```diff
@@ -282,3 +289,36 @@ def read_header(self):
         self.field_offsets = field_offsets
         self.field_types = _pfields
         fd.close()
+
+
+class SinkParticleFileHandlerCsv(ParticleFileHandler):
+    """Handle sink files from a csv file, the format from the sink particle in ramses"""
+
+    ptype = "sink_csv"
+    fname = "sink_{iout:05d}.csv"
+    file_descriptor = None
+    config_field = "ramses-sink-particles"
+    reader = _ramses_particle_csv_file_handler
+    attrs = (("nsink", 1, "i"), ("nindsink", 1, "i"))
+    second_read = False
+
+    def read_header(self):
+        if not self.exists:
+            self.field_offsets = {}
+            self.field_types = {}
+            self.local_particle_count = 0
+            return
+        field_offsets = {}
+        _pfields = {}
+
+        fields, self.local_particle_count = _read_part_csv_file_descriptor(self.fname)
+
+        for ind, field in enumerate(fields):
+            field_offsets[self.ptype, field] = ind
+            _pfields[self.ptype, field] = "d"
+
+        self.field_offsets = field_offsets
+        self.field_types = _pfields
+
+    def sec_read(self):
+        pass
```

Comment:
I think this got simplified into oblivion by formatters, and it didn't break anything because this method isn't actually used, so might as well drop it ?
```suggestion
```